packages/Manager/src/main/resources/application-prod.yml

```diff
@@ -11,6 +11,15 @@ spring:
         mechanism: SCRAM-SHA-512
         jaas:
           config: org.apache.kafka.common.security.scram.ScramLoginModule required username="${KAFKA_SASL_USERNAME}" password="${KAFKA_SASL_PASSWORD}";
+management:
+  endpoints:
+    enabled-by-default: false
+  web:
+    exposure:
+      include: "health"
+  endpoint:
+    health:
+      enabled: true
```

Comment:
What do these changes do?

Reply:
Ah so the net result was to turn on all available monitoring in Dev, but then required setting it back to only the health endpoint in prod.